output/print_methods.go

```diff
@@ -2,59 +2,74 @@ package output
 
 import (
 	"fmt"
+	"net/http"
 	"strings"
 
 	"github.com/fatih/color"
 	"github.com/visola/go-http-cli/request"
 )
 
-type bodyBuffer struct {
-	*strings.Reader
-}
-
-func (bb *bodyBuffer) Close() error {
-	return nil
-}
-
 // PrintRequest outputs the http.Request
-func PrintRequest(request request.Request) error {
+func PrintRequest(request request.Request) {
 	boldGreen := color.New(color.Bold, color.FgGreen).PrintfFunc()
 	boldGreen("\n%s %s\n", request.Method, request.URL)
 
-	sentHeaderKeyColor := color.New(color.Bold, color.FgBlack).PrintfFunc()
-	sentHeaderValueColor := color.New(color.FgBlack).PrintfFunc()
+	printHeaders(request.Headers)
+	printCookies(request.Cookies)
+	printBody(request.Body, ">>")
+}
 
-	for headerName, values := range request.Headers {
-		sentHeaderKeyColor("%s:", headerName)
-		if len(values) > 1 {
-			for _, val := range values {
-				sentHeaderValueColor("\n  %s", val)
-			}
-			fmt.Println("")
-		} else {
-			sentHeaderValueColor(" %s\n", values[0])
+// PrintResponse outputs a http.Response
+func PrintResponse(response request.Response) {
+	if response.StatusCode < 300 {
+		color.Green("%s %s\n", response.Status, response.Protocol)
+	} else if response.StatusCode < 400 {
+		color.Yellow("%s %s\n", response.Status, response.Protocol)
+	} else {
+		color.Red("%s %s\n", response.Status, response.Protocol)
+	}
+
+	printHeaders(response.Headers)
+	printBody(response.Body, "<<")
+}
+
+func printBody(body string, linePrefix string) {
+	if body != "" {
+		bodyColor := color.New(color.Bold).PrintfFunc()
+		split := strings.Split(body, "\n")
```

Comment:
If you decide to keep this variable, maybe `lines` is more appropriate (especially since you call it `line` in the `for` loop below).

Reply:
What happens if the response body looks something like
```json
{
	"pullRequestId": 58,
    "commentId": 5,
    "text": "foo\n bar\n baz\n"
}
```

Reply:
Then it won't look like that. It would look like this:
```
{
    "pullRequestId": 58,
    "commentId": 5,
    "text": "foo
bar
baz
"
}
```
And it would be a broken JSON. Either way, it would be split and printed correctly.

Reply:
For future reference:
https://play.golang.org/p/aaOFomggG8B
and 
https://jsbin.com/jijicucovi/edit?js,console